frontend/src/Context/PreviewPublisherProvider/usePreviewPublisher/usePreviewPublisher.tsx

```diff
@@ -212,15 +212,29 @@ const usePreviewPublisher = (): PreviewPublisherContextType => {
       return;
     }
 
-    publisherRef.current = initPublisher(undefined, { insertDefaultUI: false }, (err: unknown) => {
-      if (err instanceof Error) {
-        publisherRef.current = null;
-        if (err.name === 'OT_USER_MEDIA_ACCESS_DENIED') {
-          console.error('initPublisher error: ', err);
+    const videoFilter: VideoFilter | undefined = localBlur
+      ? {
+          type: 'backgroundBlur',
+          blurStrength: 'high',
+        }
+      : undefined;
+
+    publisherRef.current = initPublisher(
+      undefined,
+      { insertDefaultUI: false, videoFilter },
+      (err: unknown) => {
+        if (err instanceof Error) {
+          publisherRef.current = null;
+          if (err.name === 'OT_USER_MEDIA_ACCESS_DENIED') {
+            console.error('initPublisher error: ', err);
+          }
         }
       }
-    });
+    );
     addPublisherListeners(publisherRef.current);
+    // the local blur value gets rendered a few times which can cause this to be rendered a few times
+    // excluding from the dependencies to avoid that
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [addPublisherListeners]);
```

Comment:
🤔 maybe you can wrap the videoFilter in a `useMemo` instead?

Reply:
that's. what I thought about doing first but then decided using `useMemo` for something little like that might be too much 🤔 but now since you brought it up, I'll change it to have `useMemo` 🙏

Reply:
on the second note, after messing with `useMemo`s for a bit, I think maybe doing the comment and excluding it from the `deps` here would be a better way. I'm seeing some inconsistent behavior with publisher destroys on blur toggle as well as the value not carrying over from the meeting room. If you're okay @maikthomas I'll keep this as-is 🙏

Reply:
It looks like the initial state and the current state for the user's blur settings need to be decoupled since we don't want the current state of blur to trigger a re-render. We could set the initial blur state to a ref so it doesn't trigger a re-render and since it's not going to change. That'll make this callback stable.

Reply:
I think we can simplify it, I was thinking two-lines, but I might be wrong 😅  

Instead of: `const videoFilter: VideoFilter | undefined = defaultSettingsBlurRef`, could you try: `const videoFilter: VideoFilter | undefined = defaultSettingsBlurRef.current`?

We'd still want the `localBlur` to be initialized to the value from `user.defaultSettings`, so keep it what you had before.

We can pair up too, we've got time before standup lol

Reply:
done for both requests! for some reason mocking `hasMediaProcessorSupport` took me a second but it was super easy in the end 😂